RHDEVDOCS-5247: Documentation for Pipelines Resolvers (#60482)
openshift/openshift-docs: modules/op-resolver-git-config-anon.adoc

// This module is included in the following assembly:
//
// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc

:_content-type: PROCEDURE
[id="resolver-git-config-anon_{context}"]
= Configuring the Git resolver for anonymous Git cloning

> If you want to use anonymous Git cloning, you can optionally configure the default Git revision, fetch timeout, and default repository URL for pulling remote pipelines ans tasks from a Git repository.

Author: Yes, it is optional to configure them. WItyh anonymous cloning things work fine with no config - unlike the SCM API

> If you want to use anonymous Git cloning, you can optionally configure the default Git revision, fetch timeout, and default repository URL for pulling remote pipelines and tasks from a Git repository.

Reviewer: ```suggestion
If you want to use anonymous Git cloning, you can configure the default Git revision, fetch timeout, and default repository URL for pulling remote pipelines and tasks from a Git repository.
```
again, "can" rather than "must" implies it's optional

.Procedure

. To edit the `TektonConfig` custom resource, enter the following command:
+
[source,terminal]
----

> $ oc edit TektonConfig config

Author: I'd like to keep what we have for consistency with other places in the same Pipelines doc

----

> +

Reviewer: ```suggestion

```
not needed between steps generally

> . In the `TektonConfig` custom resource, specify configuration in the `pipeline.git-resolver-config` spec:

Reviewer: ```suggestion
. In the `TektonConfig` custom resource, edit the `pipeline.git-resolver-config` spec:
```

> . In the `TektonConfig` custom resource, in the `pipeline.git-resolver-config` spec, specify  configuration:

Reviewer 2: Seems like you're missing some words here with "specify configuration". Should it be "specify the following configuration" ?  (multiple files)
